packages/reactive-dict/reactive-dict.js

```diff
@@ -164,6 +164,23 @@ _.extend(ReactiveDict.prototype, {
     
   },
 
+  delete: function(key) {
+    var self      = this;
+    var oldKeys   = self.keys;
+    var didRemove = false;
+
+    if (_.has(oldKeys, key)) {
+      changed(self.keyDeps[key]);
```

Comment:
It feels like most of this method's code could be turned into a call to `this.set(key, undefined)`.